src/ProjectSystem/Impl/Project/FileSystemMirroringProjectFileGenerator.cs

```diff
@@ -63,6 +64,7 @@ public abstract class FileSystemMirroringProjectFileGenerator : IVsProjectGenera
                             new XProjElement("UserProperties")
                         )
                     ),
+                    new XProjElement("Target", new XAttribute("Name", "Build")),
```

Comment:
Yes, we'll need to doc the manual workaroud (add `<Target Name="Build"/>`) for now. Later we can do the same thing PTVS did, and implement project upgrade, but that's a can of worms that I really don't want to open right now.

And yes, this makes both Start commands being unavailable.